docs/create-a-neuron.md

```diff
@@ -37,12 +39,15 @@ dfx --identity neuron-hotkey identity get-principal
 ```
 Example output: `wuyst-x5tpn-g5wri-mp3ps-vjtba-de3xs-w5xgb-crvek-tucbe-o5rqi-mae`
 
-**Note:** This hotkey is used for NNS proposal submissions only.
+## (Optional) Hardware wallet identity
+
+In the NNS UI you can also add a hardware wallet as a controlling entity. You will not be able to submit proposals with the hardware wallet entity though. But you can keep all your funds on the hardware wallet.
 
 ### Create and Manage Neuron via NNS Frontend Dapp and Internet Identity
```

Comment:
I would have expected this to appear before `## Create a neuron Hotkey`. Also, at the moment, it mentions the hardware wallet. Maybe you can just remove that and add the section `## (Optional) Hardware wallet identity` at the very end as a "advanced" topic.

Reply:
Agreed, let me make the change